integration_tests/integration/court-register-amend.spec.ts

```diff
@@ -60,31 +62,31 @@ context('Court register - amend existing court', () => {
       IndexPage.verifyOnPage().courtRegisterLink().click()
       AllCourtsPagedPage.verifyOnPage()
         .viewCourtLink('SHFMC')
-        .should('contain.text', 'Sheffield Magistrates Court')
```

Comment:
Mostly refactor so the assertions done on constants that magic numbers, new test at end